nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -148,22 +129,88 @@ def wrapped(*args, **kwargs):
 
 
 def wrap_module_call(module_call):
+    from nncf.torch.dynamic_graph.patch_pytorch import ORIGINAL_OPERATORS
+    NAMES_ORIGINAL_OPERATORS = [op.name for op in ORIGINAL_OPERATORS]
+
     def wrapped(self, *args, **kwargs):
         ctx = get_current_context()
         if not ctx or self.__class__ in _IGNORED_SCOPES:
             if isinstance(self, DataParallel):
                 _warn_data_parallel()
             return module_call(self, *args, **kwargs)
         ctx.push_scope(self)
-        retval = module_call(self, *args, **kwargs)
-        if type(self).__name__ in ITERATION_MODULES.registry_dict:
+        is_nncf_layer = isinstance(self, _NNCFModuleMixin)
+        if is_nncf_layer:
+            op_name = self.op_func_name
+        else:
+            op_name = self.__class__.__name__
+        is_layer_or_op = op_name in NAMES_ORIGINAL_OPERATORS or is_nncf_layer
+        op_address = ctx.get_caller_context(op_name)
+        str_op_address = str(op_address)
+        if ctx.elastic_depth and is_layer_or_op and ctx.in_skipped_block:
+            ctx.register_operator_call(op_address.operator_name, op_address.scope_in_model)
+            retval = ctx.tensor_cache
+            if str_op_address in ctx.end_node_name_of_skipped_block:
+                assert ctx.in_skipped_block is True
+                ctx.in_skipped_block = False
+            if str_op_address in ctx.start_node_name_of_skipped_block:
+                assert ctx.in_skipped_block is False
+                ctx.in_skipped_block = True
+        else:
+            retval = module_call(self, *args, **kwargs)
+
+        if type(self).__name__ in ITERATION_MODULES.registry_dict.keys():
             ctx.reset_operator_call_count_in_scope(ctx.scope)
         ctx.pop_scope()
         return retval
 
     return wrapped
 
 
+def _process_ordinary_op(op_address: 'OperationAddress',
```

Comment:
`_execute_op` could be a more fitting name.

Reply:
Thanks! Fixed